packages/flutter/lib/src/material/feedback.dart

```diff
@@ -0,0 +1,73 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+
+/// Provides platform-specific acoustic and/or haptic feedback for certain
+/// actions.
+///
+/// See also:
+/// * [FeedbackWrapper] to provide feedback for a gesture callback by
+///   wrapping it.
+class Feedback {
+  Feedback._();
+
+  /// Provides platform-specific feedback for a tap.
+  static Future<Null> forTap(BuildContext context) async {
+    switch(_platform(context)) {
+      case TargetPlatform.android:
+      case TargetPlatform.fuchsia:
+        return SystemSound.play(SystemSoundType.click);
+      default:
+        return new Future<Null>.value();
+    }
+  }
+
+  /// Provides platform-specific feedback for a long press.
+  static Future<Null> forLongPress(BuildContext context) {
+    switch(_platform(context)) {
+      case TargetPlatform.android:
+      case TargetPlatform.fuchsia:
+        return HapticFeedback.vibrate();
+      default:
+        return new Future<Null>.value();
+    }
+  }
+
```

Comment:
Can we merge these classes and far `Feedback.forTap(context)` and `Feedback.wrapperForTap(callback, context)` ?

Having two classes with two static methods seems like it will lead to confusion.